diff --git a/Alderis/ColorPickerInnerViewController.swift b/Alderis/ColorPickerInnerViewController.swift
index dcfb790..dc60623 100644
--- a/Alderis/ColorPickerInnerViewController.swift
+++ b/Alderis/ColorPickerInnerViewController.swift
@@ -28,7 +28,7 @@ internal class ColorPickerInnerViewController: UIViewController {
 	let configuration: ColorPickerConfiguration
 	var color: Color
 
-	var tab: ColorPickerTab {
+	var selectedColorTab: ColorPickerTab {
 		get { configuration.visibleTabs[currentTab] }
 		set { currentTab = configuration.visibleTabs.firstIndex(of: newValue) ?? 0 }
 	}
@@ -286,7 +286,7 @@ internal class ColorPickerInnerViewController: UIViewController {
 		])
 
 		colorDidChange()
-		tab = configuration.initialTab
+		selectedColorTab = configuration.initialTab
 		tabsView?.selectedSegmentIndex = currentTab
 	}
 
